app/services/moves/sweeper.rb

```diff
@@ -36,6 +36,8 @@ def update_nomis_event_ids!
         date: date,
         from_location_id: locations.map(&:id)
       )
+
+      update_nomis_event_ids_when_duplicate(scope)
```

Comment:
Are we using the same logic to detect potential duplicates as the unique index? Should this (L35-38 above) also have to_location and time_due?

Reply:
The whole sweeper has been built around the concept that from_location + date + person = same move, this is the logic that was there before and the spec. Eventually using a timestamp could be a very bad idea, it normally gives a lot of problems.
If you guys think the to_location is relevant so be it

Reply:
Hi @ngw, from the spec doc:

> A Person could have multiple Moves on the same date, given they were for a different purpose (eg. a Move to court in the morning, and a Hospital Move in the evening).

It's important the uniqueness validation matches this logic, so I'll add a note on this PR next to the schema